clang/test/OpenMP/reduction_complex.c

```diff
@@ -0,0 +1,1071 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --function-signature --include-generated-funcs --replace-value-regex "__omp_offloading_[0-9a-z]+_[0-9a-z]+" "reduction_size[.].+[.]" "pl_cond[.].+[.|,]" --prefix-filecheck-ir-name _
```

Comment:
Add a filter here to restrict the check lines to the functions we care about.

Reply:
I've removed the checks which didn't pertain to the complex reduction section of the code.